WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/variations/attributes/AddAttributeTermsViewModel.kt

```diff
@@ -0,0 +1,57 @@
+package com.woocommerce.android.ui.products.variations.attributes
+
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.asLiveData
+import com.woocommerce.android.model.ProductAttributeTerm
+import com.woocommerce.android.viewmodel.ScopedViewModel
+import com.woocommerce.android.viewmodel.getStateFlow
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class AddAttributeTermsViewModel @Inject constructor(
+    savedState: SavedStateHandle,
+    private val termsListHandler: AttributeTermsListHandler
+) : ScopedViewModel(savedState) {
+    private val termsFlow = savedState.getStateFlow(
+        this,
+        emptyMap<Int, ProductAttributeTerm>()
+    )
+    val termsListState = termsFlow
+        .map { it.map { (_, term) -> term } }
```

Comment:
np: we can change  it.map { (_, term) -> term } -> it.values.toList()
```suggestion
        .map { it.values.toList() }
```

Reply:
Good one! Thanks! Done in https://github.com/woocommerce/woocommerce-android/pull/9748/commits/d62aff9c8d73632e4a2d813c64359e42eece97ae.